packages/components/organisms/f-header/package.json

```diff
@@ -40,12 +40,13 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-services": "1.3.0",
-    "@justeat/f-vue-icons": "1.4.0"
+    "@justeat/f-services": "1.3.0"
   },
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.1.1",
-    "@justeat/f-trak": ">=0.6.0"
+    "@justeat/f-trak": ">=0.6.0",
+    "@justeat/f-vue-icons": "2.3.0",
```

Comment:
`f-vue-icons` is likely still a dependency rather than a `peerDependency` as it will include the icons at build time?

Reply:
May even not need to be included here, as it would be covered by the mono-repo dependencies due to it being run at build time?

Reply:
Oh, I wanted to move it to dev dependencies

Reply:
dev deps should be fine as consuming applications won't need to the icons packages as it's already bundled up.